app/assets/javascripts/range_limit_config.js

```diff
@@ -0,0 +1,12 @@
+Blacklight.onLoad(function() {
+  var dateRangeFacets = [
+    $('.blacklight-cho_date_range_norm_isim'),
+    $('.blacklight-cho_date_range_hijri_isim')
+  ];
+
+  for(i = 0; i < dateRangeFacets.length; i++) {
+    dateRangeFacets[i].data('plot-config', {
```

Comment:
I'm thinking we might want to do something along this line https://github.com/flot/flot/blob/master/API.md#customizing-the-legend to also support #895 ?

Reply:
I'm not sure I understand the ask.

Reply:
Not an ask... just started thinking about that future ticket.